docs/sources/introduction/_index.md

```diff
@@ -20,9 +20,9 @@ After you have [installed Grafana]({{< relref "../setup-grafana/installation" >}
 
 The following sections provide an overview of Grafana features and links to product documentation to help you learn more. For more guidance and ideas, check out our [Grafana Community forums](https://community.grafana.com/).
 
-## Explore metrics, logs, and traces
+## Explore metrics, logs, traces and profiles
```

Comment:
Each of the Explore apps is a product/feature name, which means they should be proper nouns. i'm not sure if we can use proper nouns in a list like this. I'll ask the other writers for input.

Reply:
```suggestion
## Explore metrics, logs, traces, and profiles
```